localized_fields/fields/field.py

```diff
@@ -199,36 +206,25 @@ def validate(self, value: LocalizedValue, *_):
         if self.null:
             return
 
-        primary_lang_val = getattr(value, settings.LANGUAGE_CODE)
+        for lang in self.required:
+            lang_val = getattr(value, settings.LANGUAGE_CODE)
 
-        # NOTE(seroy): use check for None, instead of `not primary_lang_val`
-        # condition, cause in this way we can not save '' value
-        if primary_lang_val is None:
-            raise IntegrityError(
-                'null value in column "%s.%s" violates not-null constraint' % (
-                    self.name,
-                    settings.LANGUAGE_CODE
-                )
-            )
+            # NOTE(seroy): use check for None, instead of `not lang_val`
+            # condition, cause in this way we can not save '' value
+            if lang_val is None:
+                raise IntegrityError('null value in column "%s.%s" violates' \
+                                     'not-null constraint' % (self.name, lang))
 
     def formfield(self, **kwargs):
         """Gets the form field associated with this field."""
 
         defaults = {
             'form_class': LocalizedFieldForm
         }
-
+        if issubclass(kwargs.get('form_class', LocalizedFieldForm),
+                      LocalizedFieldForm):
+            defaults.update({
+                'required_langs': self.required
+            })
```

Comment:
This is a moot point, but right now we have `LocalizedAutoSlugField` which set `form_class` as `CharField`, and `CharField` is not expects `required_langs` parameter.